src/middlewared/middlewared/plugins/pool.py

```diff
@@ -4182,6 +4182,131 @@ def __handle_zfs_set_property_error(self, e, properties_definitions):
             raise CallError(pjob.error)
         return data
 
+    @accepts(
+        Str('id', required=True),
+        Str('type', enum=['USER', 'GROUP']),
+        Ref('query-filters'),
+        Ref('query-options'),
+    )
+    @item_method
+    async def get_userquota(self, id, type, filters, options):
+        """
+        Returns a list of the specified `type` of  quotas on the ZFS dataset `id`.
```

Comment:
Use present tense (s/Returns/Return/).  It's a small thing in this one, but makes this sort of thing easier to read.  More instances below.